incubator/sentry-kubernetes/templates/deployment.yaml

```diff
@@ -34,4 +34,4 @@ spec:
       tolerations:
 {{ toYaml .Values.tolerations | indent 8 }}
     {{- end }}
-      serviceAccountName: {{ if .Values.rbac.create }}{{ template "sentry-kubernetes.fullname" . }}{{ else }}"{{ .Values.rbac.serviceAccountName }}"{{ end }}
+      serviceAccountName: {{ if .Values.rbac.create }}{{ template "sentry-kubernetes.serviceAccountName" . }}{{ else }}"{{ .Values.rbac.serviceAccountName }}"{{ end }}
```

Comment:
Just this:
`serviceAccountName: {{ template "sentry-kubernetes.serviceAccountName" . }}`

Reply:
fixed! thanks again

Reply:
@unguiculus any other feedback?